tests/acceptance/conftest.py

```diff
@@ -28,7 +28,7 @@ def pytest_configure(config):
             last_built = int(time.time()) - data["built"]
 
             if last_built <= 3600:
-                print (
```

Comment:
It was actually `black` that autoformatted this, but I guess it isn't enough to detect that that `__future__.print_function` was imported.

Also lol, TIL this works on 2.x and 3.x:

```
>>> print  ('asdf')
asdf
>>> print ('asdf')
asdf
>>> print('asdf')
asdf
```